docs/data/stashcache/install-cache.md

```diff
@@ -5,192 +5,217 @@ This document describes how to install a StashCache cache service.  This service
 network to cache data frequently used on the OSG, reducing data transfer over the wide-area network and
 decreasing access latency.
 
-!!! note
-    The _cache_ service must be registered with the OSG if it is to be used by clients. You may start 
-    registration procedure prior to finishing the installation by [using this link](#registering-the-cache) 
-    along with the basic information like:
-
-    * Resource name and hostname.
-    * Administrative and security contact.
 
-## Installation prerequisites for the Cache
+Before Starting
+---------------
 
-Before starting the installation process, consider the following mandatory points:
+Before starting the installation process, consider the following requirements:
 
-* __Operating system:__ Only RHEL 7 and compatible operating systems are supported.
+* __Operating system:__ A RHEL 7 or compatible operating systems.
 * __User IDs:__ If they do not exist already, the installation will create the Linux user IDs `condor` and
   `xrootd`
-* __Host certificate:__ The StashCache server uses a host certificate to advertise to a central collector.
-  The [host certificate documentation](/security/host-certs.md) provides more information on setting up host
-  certificates.
-* __Network ports:__ The cache service requires inbound ports, one for the xrootd protocol, one for
-  HTTP and, if authenticated StashCache is used, one for HTTPS.  The defaults are:
-    * TCP port 1094 for xrootd.
-    * TCP port 8000 for HTTP.
-    * TCP port 8443 for HTTPS (optional).
-* __Hardware requirements:__ We recommend that a StashCache server has at least 10Gbps connectivity, 1TB of
+* __Host certificate:__ Required for reporting and authenticated StashCache.
+  (Authenticated StashCache is an optional feature.)
```

Comment:
A whole sentence in a parenthetical that's not part of another sentence looks strange:

```suggestion
  Authenticated StashCache is an optional feature.
```